perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -463,11 +466,8 @@ sub _missed_collection_reports {
             = $report->get_extra_field_value('service_item_name') // '';
         next if $missed_collection_reports{$service_item_name};
 
-        my $latest_comment
-            = $report->comments->search(
-                {},
-                { order_by => { -desc => 'id' } },
-        )->first;
+        my $status = $ws->{WorksheetStatusName};
```

Comment:
This will always be Open by the if in line 452, that right?

Reply:
No, that was not right! Clearly wasn't thinking straight yesterday.

Reply:
Have pushed a fixup commit which uses the fms_state matched to the report->state to determine which text to display.